.github/workflows/pull_request.yml

```diff
@@ -24,6 +24,10 @@ jobs:
         run: |
           make pb
 
+      - name: Run the Golang unit tests
+        run: |
+          make test
+
```

Comment:
Whether having this here or in a separate job is open for debate. Separate is probably faster since GH actions run in parallel. Can be addressed later.

Reply:
+1 for separate jobs, definitely.

Reply:
I have a separate workflow file, somewhere in this branch's history

Reply:
check df027ad

Reply:
> check [df027ad](https://github.com/omec-project/upf-epc/commit/df027add4601b0e43a036bf7d627d59f5f6b0850)

seems to work as well. can you remove this snippet here?